package.json

```diff
@@ -67,7 +67,7 @@
     "test:e2e:stream": "mocha \"out/**/*.e2e.js\" -s 0 --grep \"\\(stream\\)\"",
     "_build:compile": "npm run clean && npm run compile",
     "build": "npm run _build:compile && npm run lint && npm test",
-    "watch": "npm run _build:compile -- --sourceMap --watch",
+    "watch": "npm run _build:compile -- -- --sourceMap --watch",
```

Comment:
This change shouldn't happen I guess?

Reply:
Oh nm, TIL about the double-double dashes :)

Reply:
Unfortunately, this is an issue that was already present in the trunk. I have fixed it in the separated commit.